cmd/root.go

```diff
@@ -178,6 +178,13 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&rest.Domain, "domain", defaultDomain, "Your assigned Akita domain (e.g. company.akita.software)")
 	rootCmd.PersistentFlags().MarkHidden("domain")
 
+	// Use a proxy or permit an unknown certificate.
+	rootCmd.PersistentFlags().StringVar(&rest.UseProxy, "proxy", "", "The domain name or IP address of an HTTP proxy server to use")
+	rootCmd.PersistentFlags().BoolVar(&rest.PermitInvalidCertificate, "skip-validate", false, "Skip TLS validation on the connection to api.akita.software")
+	rootCmd.PersistentFlags().MarkHidden("skip-validate")
```

Comment:
Rename to `skip-tls-validate` or `skip-tls-validation`?

Reply:
Changed in https://github.com/akitasoftware/akita-cli/pull/195/commits/3a73809e34f771b0dc7149c89e0b705710b66d1c